PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetProperties.h

```diff
@@ -0,0 +1,233 @@
+#ifndef ALIANALYSISTASKEMCALJETPROPERTIES_H
+#define ALIANALYSISTASKEMCALJETPROPERTIES_H
+/**
+ * \file AliAnalysisTaskEmcalJetProperties.h
+ * \brief Declaration of class AliAnalysisTaskEmcalJetProperties
+ *
+ * In this header file the class AliAnalysisTaskEmcalJetProperties is declared.
+ * This is a sample task that shows how to write a simple user analysis task
+ * using the EMCal jet framework. It is also used to do automatic benchmark
+ * tests of the software.
+ *
+ * \author Salvatore Aiola <salvatore.aiola@cern.ch>, Yale University
```

Comment:
Please fix the author

Reply:
This code is the modified version of the previous code AliAnalysisTaskJetProperties.h.
Are you suggesting keeping our names?

Reply:
This task is inherited from the sample task. We have modified and implemented our task. Our task is also the modified version of  AliAnalysisTaskJetProperties.h, which was the code of Sidharth Kumar Prasad. We have now recently modified this code. So I have changed the author's name accordingly.

Reply:
Since it sounds like it was unclear, I'll be precise - remove Salvatore, and in (please adapt the emails as needed, or you can omit the field `<...>` if you don't want to put your email address:

```
\author Sidharth Kumar Prasad <another_email@address.com>, Bose Institute
\author Prottoy Das <an_email@address.com>, Bose Institute
\authro Debjani Banerjee <your_email@address.com>, Bose Institute
```

Reply:
I have added the author names